crates/esplora/tests/async_ext.rs

```diff
@@ -19,14 +20,21 @@ struct TestEnv {
 impl TestEnv {
     fn new() -> Result<Self, anyhow::Error> {
         let bitcoind_exe =
-            bitcoind::downloaded_exe_path().expect("bitcoind version feature must be enabled");
-        let bitcoind = BitcoinD::new(bitcoind_exe).unwrap();
+            bitcoind::exe_path().expect("Cannot find bitcoind daemon, set BITCOIND_EXEC environment variable with the path to bitcoind");
+        let mut bitcoind_conf = bitcoind::Conf::default();
+        bitcoind_conf.p2p = bitcoind::P2P::Yes;
+        let bitcoind = BitcoinD::with_conf(&bitcoind_exe, &bitcoind_conf)?;
 
         let mut electrs_conf = Conf::default();
         electrs_conf.http_enabled = true;
-        let electrs_exe =
-            electrsd::downloaded_exe_path().expect("electrs version feature must be enabled");
-        let electrsd = ElectrsD::with_conf(electrs_exe, &bitcoind, &electrs_conf)?;
+        let electrs_exe = electrsd::exe_path().expect("Cannot find electrs daemon, set ELECTRS_EXEC environment variable with the path to electrs");
+        let electrsd = ElectrsD::with_conf(&electrs_exe, &bitcoind, &electrs_conf)?;
+
+        // Alive checks
+        let _ = bitcoind.client.ping().unwrap(); // without using bitcoind, it is dropped and all the rest fails.
+        let _ = electrsd.client.ping().unwrap();
+        assert!(bitcoind.client.ping().is_ok());
+        assert!(electrsd.client.ping().is_ok());
```

Comment:
@RCasatta any ideas why I am hitting
https://github.com/RCasatta/electrsd/blob/9f6508ddad8dcf15cccfcefd53597145f31d46c7/src/lib.rs#L268

in this test:
```shell
crates-test> test simple_test ... FAILED(B
crates-test> test test_update_tx_graph_without_keychain ... FAILED(B
crates-test> failures:
crates-test> ---- simple_test stdout ----
crates-test> Error: Remaining attempts 2
crates-test> Caused by:
crates-test>     0: Remaining attempts 1
crates-test>     1: Remaining attempts 0
crates-test>     2: EarlyExit(ExitStatus(unix_wait_status(256)))
```

I am using the `electrs` crate from https://github.com/RCasatta/electrsd/pull/72